spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -517,7 +517,7 @@
         let(:dst_cloud_network_2) { FactoryGirl.create(:cloud_network) }
         let(:dst_flavor) { FactoryGirl.create(:flavor) }
         let(:dst_security_group) { FactoryGirl.create(:security_group) }
-        let(:conversion_host_vm) { FactoryGirl.create(:vm, :ext_management_system => dst_ems) }
+        let(:conversion_host_vm) { FactoryGirl.create(:vm_openstack, :ext_management_system => dst_ems, :cloud_tenant => dst_cloud_tenant) }
```

Comment:
Can we assign a name to this `:cloud_tenant` and have an `expect` statement to match it against the `os_project_name`?  My thought is that: If this had been there, we would have detected this bug early on.

Reply:
Done.